### PR TITLE
docs(store): remove the extra function call

### PR DESCRIPTION
**Pull request by santoshyadav198613**
The function calls work now, so removing extra function call

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Closes #

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

**Comment by brandonroberts** [comment]
We need to have an additional note here that says this is if you have Ivy enabled, and if you're using ViewEngine, you'll need a wrapper function.

**Comment by santoshyadav198613** [comment]
> We need to have an additional note here that says this is if you have Ivy enabled, and if you're using ViewEngine, you'll need a wrapper function.

I was thinking about it, thanks will do the same.

**Comment by santoshyadav198613** [comment]
Hi @brandonroberts,
Let me know if it is fine now.

File: modules/store/src/reducer_creator.ts [hunk 1]
```diff
@@ -202,9 +202,8 @@ export function on(
  * @usageNotes
  *
  * - Must be used with `ActionCreator`'s (returned by `createAction`).  Cannot be used with class-based action creators.
- * - An action can be associated with multiple state change functions. In this case the functions will be executed in the specified order.
```
Comment (brandonroberts) [inline modules/store/src/reducer_creator.ts:205]: This one should not be removed

File: modules/store/src/reducer_creator.ts [hunk 2]
```diff
@@ -202,9 +202,8 @@ export function on(
...
- * - The returned `ActionReducer` should additionally be returned from an exported `reducer` function.
- * This is because [function calls are not supported](https://angular.io/guide/aot-compiler#function-calls-are-not-supported) by the AOT compiler.
+ * - The returned `ActionReducer` should additionally be returned from an exported `reducer` function, if you are using View Engine.
+ * In case you are using Ivy the extra function `reducer` can be removed.
```
Comment (brandonroberts) [inline modules/store/src/reducer_creator.ts:206]:
```suggestion
 * In case you are using Ivy the extra function `reducer` is not required.
```

Comment (santoshyadav198613) [reply modules/store/src/reducer_creator.ts]: Done

**Comment by brandonroberts** [comment]
👏

**Pull request merged**